lib/liquid/combinators/tags/end_block.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Liquid.Combinators.Tags.EndBlock do
+  @moduledoc """
+  Verifies when block is closed and send the AST to end the block
+  ```
```

Comment:
Why are these here?

Reply:
This module generalize the close of all blocks and it marks the end of internal recursion.

Reply:
No I mean why are the "```" in the documentation?